SessionMessagingKit/Database/Models/GroupMember.swift

```diff
@@ -161,56 +161,34 @@ extension GroupMember: ProfileAssociated {
         }
         
         /// We want to sort the member list so the most important info is at the top of the list, this means that we want to prioritise
-        /// • Invite failed, sorted as NameSortingOrder
-        /// • Invite not sent, sorted as NameSortingOrder
-        /// • Sending invite, sorted as NameSortingOrder
-        /// • Invite sent, sorted as NameSortingOrder
-        /// • Invite status unknown, sorted as NameSortingOrder
-        /// • Pending removal, sorted as NameSortingOrder
-        /// • Admin promotion failed, sorted as NameSortingOrder
-        /// • Admin promotion not sent, sorted as NameSortingOrder
-        /// • Sending admin promotion, sorted as NameSortingOrder
-        /// • Admin promotion sent, sorted as NameSortingOrder
-        /// • Admin promotion status unknown, sorted as NameSortingOrder
+        /// • Current user
         /// • Admin, sorted as NameSortingOrder
         /// • Member, sorted as NameSortingOrder
-        ///
-        /// And the current user should appear at the top of their respective group
```

Comment:
We actually do want to keep this existing sort order on the "Edit Group" screen for admins (the "Group Members" screen should have it's own order)

Reply:
OK, got it, I'll change it back and create another sorting for Group Members screen.